pkg/cmd/render/render_test.go

```diff
@@ -488,3 +490,105 @@ type fakeBootstrapIPLocator struct {
 func (f *fakeBootstrapIPLocator) getBootstrapIP(ipv6 bool, machineCIDR string, excludedIPs []string) (net.IP, error) {
 	return f.ip, nil
 }
+
+const installConfigSingleStackIPv4 = `
+apiVersion: v1
+metadata:
+  name: my-cluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  machineNetwork:
+  - foo: bar
+    cidr: 10.0.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+`
+
+const installConfigDualStack = `
+apiVersion: v1
+metadata:
+  name: my-cluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
```

Comment:
What is the relationship between `networking.machineCIDR` and `networking.machineNetwork[x].cidr`?

Reply:
the former is deprecated and used only in a fallback case